### DISCUSSION TITLE
Problem Compiling ASVS

### DISCUSSION THREAD

**Discussion by JoergBruenner** [OWASP/ASVS#1316] [Q&A]
Hi,

I tried to build docker image manually. The simple make wouldn't work. I followed the instructions
in COMPILE.md.  Building the image was fine. I've seen it in the Docker desktop for windows. Running
the document builder failed:
>docker run --rm -v <base_dir>/asvs/4.0:/data asvs/documentbuilder
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/scripts/run.sh": stat /scripts/run.sh: no such file or directory: unknown.

Can you tell me please, what's wrong.

Thanks a lot for your help
Jörg

**Comment by tghosth** [comment]
We now have a new mechanism which should hopefully work...
